frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -811,6 +811,60 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).default,
     },
   },
+  // Adding this Extension because dynamic endpoint is not avbl
+  // Todo(bipuladh): Remove once SDK is mature enough to support list page
+  {
+    type: 'HorizontalNavTab',
+    properties: {
+      model: models.StorageSystemModel,
+      page: {
+        name: '%ceph-storage-plugin~Backing Store%',
```

Comment:
```suggestion
      page: {
       // t('ceph-storage-plugin~Backing Store')
        name: '%ceph-storage-plugin~Backing Store%',
```

Reply:
Add this comment on every line. It is required for i18n of strings coming from extensions.
Add below as well.